packages/core/src/task-store/workflow-ops.ts

```
@@ -362,22 +198,15 @@ export async function updateWorkflowDefinitionImpl(store: TaskStore, id: string,
-          occupantTaskIds = allOccupantTaskIds.filter((taskId) => {
-            const row = store.db.prepare(`SELECT "column" AS column FROM tasks WHERE id = ?`).get(taskId) as
-              | { column: string }
-              | undefined;
-            return row ? removedSet.has(row.column) : false;
-          });
-        }
+        
+        // FNXC:PostgresCutover 2026-06-28: async read for column check
+        const taskRows = await layer.db.select({id: schema.project.tasks.id, column: schema.project.tasks.column}).from(schema.project.tasks).where(inArray(schema.project.tasks.id, allOccupantTaskIds));
+        const colMap = new Map(taskRows.map(r => [r.id, r.column]));
+        occupantTaskIds = allOccupantTaskIds.filter(tid => {
```

ESLint

[error] 'occupantTaskIds' is never reassigned. Use 'const' instead.

(prefer-const)